packages/react-components/src/components/Telephone/Telephone.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import { CONTACTS } from './contacts';
+import { CONTACTS, contactsMap } from './contacts';
+import { patternsMap } from './patterns';
 
-export { CONTACTS };
+export { CONTACTS, contactsMap, patternsMap };
```

Comment:
Since we moved the stories into their own package, I had to do a bit of refactoring here to make sure we still have access to the same exports.